src/netbase.h

```diff
@@ -17,6 +17,10 @@
 #include <string>
 #include <vector>
 
+#ifndef WIN32
```

Comment:
Seems this include should be in `netbase.cpp`, given there's no publicly exposed reference.